local-cli/templates/HelloWorld/app.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "HelloWorld",
+  "displayName": "HelloWorld"
```

Comment:
Nope, because this is generated in the init step, where the name is all that exists.